package.json

```diff
@@ -76,6 +76,7 @@
     "babel-tape-runner": "^2.0.0",
     "benchmark": "^2.1.0",
     "common-tags": "^1.3.0",
+    "cosmiconfig": "^2.1.0",
```

Comment:
Even if this minimum version bump is necessary, this should be in the `dependencies` section.